src/setup.py

```diff
@@ -565,6 +683,91 @@ def main():
                                  "Creating backup of the original configuration")
     REPLACE_WHITELIST_CMD = CMD(COPY_CMD.format(source=RECOMMENDED_WHITELIST, target=DEFAULT_PLUGIN_CONFIGURATION_DIR), "Replacing whitelist configuration")
 
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description='Script for custom installation process for collectd AWS CloudWatch plugin'
+    )
+    parser.add_argument(
+        '-i', '--non_interactive', required=False,
```

Comment:
Agreed, "-I" (uppercase "i") or "-n" is more conventional for non-interactive.
The uppercase being considered as a negation.